### ISSUE TITLE
biber doesn’t find .bib file with non-ASCII filename due to unicode decomposition

### ISSUE THREAD

**Issue by cruessler** [plk/biber#156]
In the following MWE biber can’t find the bibliography because of the filename containing a non-ASCII character.

```
\documentclass{article}

\begin{filecontents}{ä.bib}
@book{key,
  author = {Name},
  title = {Title}
}
\end{filecontents}

\usepackage[backend=biber]{biblatex}
\addbibresource{ä.bib}

\begin{document}
\cite{key}
\printbibliography
\end{document}
```

Output of `biber main`:
```
INFO - This is Biber 2.7
INFO - Logfile is 'main.blg'
INFO - Reading 'main.bcf'
INFO - Found 1 citekeys in bib section 0
INFO - Processing section 0
INFO - Looking for bibtex format file 'ä.bib' for section 0
Wide character in print at /tmp/par-6368726973746f7068/cache-3db21175cb167b098210142012a95f034f5cd6c7/inc/lib/Log/Log4perl/Appender/Screen.pm line 41.
ERROR - Cannot find 'ä.bib'!
INFO - ERRORS: 1
```

Apparently, this is due to unicode decomposition of the filename. The filename biber is looking for contains a decomposed character (ä = U+0061 + U+0308) while the original filename in my example uses a composed character (ä = U+00e4).

The error surfaces at https://github.com/plk/biber/blob/v2.7/lib/Biber/Utils.pm#L101 in `locate_biber_file` where biber checks for the .bib file’s existence via `-e $filename` which fails.

The `bcf:datasource` tag in `main.bcf` contains the original filename.

The error is present in 2.8 (beta), too.

**Comment by plk** [comment]
I have put a fix into 2.8 beta but can't test it currently - can you see if this fixes the issue for you?

**Comment by cruessler** [comment]
I just tested the current dev branch and found everything working. Thanks for the quick fix!

(I only tested my specific use case, but I assume the fix is general enough to work in all cases comparable.)
